### PR TITLE
chore: remove `@mui/system` dependency

### PR DESCRIPTION
**Pull request by a-m-dev**
just a small change from @mui/system/styles -> @mui/material/styles in our imports + removing it from pkg.json file

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2358,7 +2358,7 @@
     "@babel/runtime" "^7.20.1"
     prop-types "^15.8.1"
 
-"@mui/system@5.11.12", "@mui/system@^5.11.12":
+"@mui/system@^5.11.12":
```
Comment (haxxmaxx) [inline yarn.lock:2361]: I think if we redo yarn.lock from scratch we shouldn't see mui/system in here at all

**Review by haxxmaxx — COMMENTED** [review]
I thought we would get complaints that the props in the styled components aren't typed. Do you know why not?

**Comment by a-m-dev** [comment]
Both are same functions, and according to the docs the only difference is in using the default theme if there is none available through context. So I think should not be any problems in types.

File: yarn.lock (hunk 1 above)
Comment (a-m-dev) [reply yarn.lock:2361]: I think `@mui/material` depends on it 👇

Screenshot 2023-03-21 at 10 40 11

**Comment by haxxmaxx** [comment]
> Both are same functions, and according to the docs the only difference is in using the default theme if there is none available through context. So I think should not be any problems in types.

hmm ok then this might not affect bundle size. I still don't understand why typescript doesn't complain about the untyped callbacks in the styled functions...

**Review by haxxmaxx — APPROVED** [review]

**Pull request merged**